diff/keys.go

```diff
@@ -0,0 +1,105 @@
+package diff
+
+import (
+	"fmt"
+	"strings"
+
+	ucfg "github.com/elastic/go-ucfg"
```

Comment:
Note: This line is equivalent to `"github.com/elastic/go-ucfg"`.

goimports normaly adds this kind of line. If `-` is used in package names, the later part of the name makes the package.